core/src/test/java/com/mygdx/game/models/BrainstormingPhaseTest.java

```diff
@@ -38,41 +38,50 @@ public void setUp() throws Exception {
 
     @Test
     public void getBrainsWithCorrectParameters(){
-        ArrayList<Brain> brainArrayList = new ArrayList<>();
-        brainArrayList.add(b1);
-        brainArrayList.add(b2);
-        bp.putIdeaOnBrainAndFire(b1, i1);
-        bp.putIdeaOnBrainAndFire(b2, i2);
-        assertEquals("Expected list with b1 and b2, but was" + bp.getBrains(), brainArrayList, bp.getBrains());
+        ArrayList<Brain> expectedBrainArrayList = new ArrayList<>();
+        expectedBrainArrayList.add(brain1);
+        expectedBrainArrayList.add(brain2);
+        brainstormingPhase.putIdeaOnBrainAndFire(brain1, idea1);
+        brainstormingPhase.putIdeaOnBrainAndFire(brain2, idea2);
+        assertEquals("Expected list with b1 and b2, but was" + brainstormingPhase.getBrains(),
+                expectedBrainArrayList.toString(), brainstormingPhase.getBrains().toString());
```

Comment:
Maybe change `b1` and `b2` according to what is returned? Also it return the same but why do you compare the toString methods?

Reply:
I forgot to change the message, and i compared the string methods because you asked me to?

Reply:
https://github.com/VildeStabell/TDT4240-Brainstorming_the_game/pull/35#discussion_r609566208

Reply:
I now see that i missread your earlier comment, since you'd asked me to compare toString() methods earlier, so i'll remove the toString() calls